erica_app/erica/elster_xml/elster_xml_tree.py

```diff
@@ -20,11 +20,12 @@ def __init__(self, name, sub_elements, is_person_specific=False, repetitions=1,
                     'E0101701', 'E0101702']
 
 # Persönliche Daten
+_ALLG_TELE_NUMMER = 'E0100008'
 _ALLG_A = ElsterXmlTreeNode(name='A', sub_elements=_PERSON_A_FIELDS)
 _ALLG_B = ElsterXmlTreeNode(name='B', sub_elements=_PERSON_B_FIELDS)
 _ALLG_BANK_INFORMATION = ElsterXmlTreeNode(name='BV', sub_elements=['E0102102', 'E0101601', 'E0102402'])
 _ALLG_VLG_ART = ElsterXmlTreeNode(name='Vlg_Art', sub_elements=['E0101201'])
-_EST1A_ALLG = ElsterXmlTreeNode(name='Allg', sub_elements=[_ALLG_A, _ALLG_B, _ALLG_VLG_ART, _ALLG_BANK_INFORMATION])
+_EST1A_ALLG = ElsterXmlTreeNode(name='Allg', sub_elements=[_ALLG_TELE_NUMMER, _ALLG_A, _ALLG_B, _ALLG_VLG_ART, _ALLG_BANK_INFORMATION])
```

Comment:
We could think about putting the `'E0100008' ` right into the array to make it axplicit it is a final node, but I might like your solution even better^^

Reply:
Thanks :D then I'll leave it as it is